rust/maprando-web/templates/common/inventory.html

```diff
@@ -0,0 +1,335 @@
+            if (ref_item == null) {
+                return;
+            }
+            // Special behaviour for pack items
+            if (ref_consumables_array.includes(ref_item)) {
+                let ref_item_num = ref_item.querySelector(".item-amount");
+                let amount = parseInt(ref_item_num.textContent);
+                if (isNaN(amount)) {
+                    amount = 0;
+                }
+                let amount_pack = 0;
+                let amount_max = 0;
+                switch (ref_item) {
+                    case ref_etanks:
+                        amount_pack = 1;
+                        amount_max = 14;
+                        break;
+                    case ref_reserves:
+                        amount_pack = 1;
+                        amount_max = 4;
+                        break;
+                    case ref_missiles:
+                        amount_pack = 5;
+                        amount_max = 255;
+                        break;
+                    case ref_supers:
+                    case ref_powerbombs:
+                        amount_pack = 5;
+                        amount_max = 50;
+                        break;
+                }
+                if (e.ctrlKey) {
+                    amount_pack *= 10;
+                }
+                // Add
+                if (e.type == "click") {
+                    amount += amount_pack;
+                    if (amount > amount_max) {
+                        amount = amount_max;
+                    }
+                }
+                // Remove
+                else if (e.type == "contextmenu") {
+                    amount -= amount_pack;
+                    if (amount < 0) {
+                        amount = 0;
+                    }
+                }
+                ref_item_num.textContent = amount;
+                ref_item_num.classList.toggle("d-none", amount == 0);
+                ref_item.classList.toggle("item-disabled", amount == 0);
+                ref_item_num.classList.toggle("item-amount-max", amount == amount_max);
+            }
+            // Majors
+            else {
+                ref_item.classList.toggle("item-disabled");
+            }
+        }
+
+        ref_inventory.addEventListener("click", onItemClick);
+        ref_inventory.addEventListener("contextmenu", onItemClick);
+    });
+
+    function getInventory() {
+        let inventory = {};
+
+        inventory.items = [];
+        for (let i in ref_item_array) {
+            inventory.items[i] = !ref_item_array[i].classList.contains("item-disabled");
+        }
+
+        function getAmount(ref_item) {
+            let amount = parseInt(ref_item.querySelector(".item-amount").textContent);
+            if (isNaN(amount)) {
+                amount = 0;
+            }
+            return amount;
+        }
+
+        inventory.max_energy = 99 + (getAmount(ref_etanks) * 100);
+        inventory.max_reserves = getAmount(ref_reserves);
```

Comment:
```suggestion
        inventory.max_reserves = getAmount(ref_reserves) * 100;
```

Reply:
@kyleb was playing around with the calculator and noticed it wasn't taking into account reserve tanks. Looks like this would be the reason why

Reply:
Yep, whoops!